pmultiqc/modules/quantms/proteobench.py

```
@@ -31,39 +37,80 @@ def get_pb_data(file_path):
         "log_vs_html": log_vs_html,
         "epsilon_html": epsilon_html,
+        "logfc_logmean_html": logfc_logmean_html,
     }
 
 
-def draw_coefficient_variation(df, plot_type):
+def draw_logmean_std_cv(
+        df,
+        plot_type,
+        runs_col=None
+    ):
```

Flake8 (7.2.0)

[error] continuation line with same indent as next logical line

(E125)